examples/coolest-todos/apps/todo-shell/next.config.js

```diff
@@ -1,13 +1,15 @@
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const { HostWebpackPlugin } = require("@leanjs/webpack");
-const withTM = require("next-transpile-modules")([
-  "@leanjs/core",
```

Comment:
@luciopalmieri do we need this?

Reply:
I'm not sure, I think that from the point of view of the example the @leanjs are third-party packages while @my-org are their own packages

Reply:
we publish `@leanjs` packages already built, so Next shoudln't need to build them again

Reply:
ok, so I was right to take it off, right?